src/backend/opencl/Array.cpp

```diff
@@ -280,7 +280,7 @@ bool passesJitHeuristics(Node *root_node) {
         // This is the maximum size of the params that can be allowed by the
         // CUDA platform.
         constexpr size_t max_nvidia_param_size = (4096 - base_param_size);
-        constexpr size_t max_amd_param_size    = (3670 - base_param_size);
+        constexpr size_t max_amd_param_size    = (3520 - base_param_size);
```

Comment:
Doesn't this change based on device ?

Reply:
We don't know what these numbers are. They don't seem to be published.

Reply:
thats unfortunate.